ha/k8s_setup/ha_setup.py

```diff
@@ -184,9 +184,12 @@ def process(self):
                          'consul_config' : {'endpoint' : consul_endpoint},
                          'event_topic' : 'hare',
                          'data_pod_label' : data_pod_label,
-                         'MONITOR' : {'message_type' : 'k8s_event', 'producer_id' : 'k8s_monitor'},
+                         'MONITOR' : {'message_type' : 'cluster_event', 'producer_id' : 'cluster_monitor'},
                          'EVENT_MANAGER' : {'message_type' : 'health_events', 'producer_id' : 'system_health',
-                                            'consumer_group' : 'health_monitor', 'consumer_id' : '1'}
+                                            'consumer_group' : 'health_monitor', 'consumer_id' : '1'},
+                         'FAULT_TOLERANCE' : {'message_type' : 'cluster_event', 'consumer_group' : 'event_listener',
+                                              'consumer_id' : '1'},
+                         'CLUSTER:NODE': {'namespace': 'cortx', 'resource_type': 'cluster:node'}
```

Comment:
Check if we can get rid of cluster:node. If yes, remove it from monitor, here and from CLUSTER:NODE as well

Reply:
Checked and tested this. We can do this change. Hence, next patch will have this change and other change wherever applicable